src/rules/enums_sorted_alphabetically.js

```diff
@@ -0,0 +1,26 @@
+import { GraphQLError } from "graphql/error";
+
+export function EnumsSortedAlphabetically(context) {
+  return {
+    EnumTypeDefinition: function EnumTypeDefinition(node, key, parent, path, ancestors) {
+      var enumValues = node.values.map(val => {
+        return val.name.value;
+      });
+
+      if (!arraysEqual(enumValues, enumValues.slice().sort())) {
+        context.reportError(
+          new GraphQLError("The enum `" + node.name.value + "` should be sorted alphabetically: " + enumValues, [
```

Comment:
Good call... yeah any more than a few enum values would make the output pretty ugly.  Changing.